docs/installing.md

```diff
@@ -0,0 +1,80 @@
+
+1. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download)
+2. Run Bitcoin-Qt in regtest mode, e.g.: `./Bitcoin-Qt -regtest -peerbloomfilters=1`
+3. Click on the *Console* tab and mine regtest bloks all at once. Run: `generatetoaddress 101 bcrt1q6j90vywv8x7eyevcnn2tn2wrlg3vsjlsvt46qz`
+
+## 5. Run Haveno
+
+Now that you have Both Bitcoin and Monero running, it's time to spin up your testing instance. You'll need to open 4 terminal windows: 1 for the seednode, 1 for the arbitrator/mediator, 1 for Alice (trader 1) and 1 for Bob (trader 2). Could be useful to rename the terminal, to have a better overview of what terminal belongs to whom (On Ubuntu: `echo -en "\033]0;NEW_TITLE\a"`).
+
+1. Run the seednode in one terminal:
+
+```
+./haveno-seednode --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=2002 --appName=haveno-XMR_STAGENET_Seed_2002 --daoActivated=false
+```
+
+2. Run the Arbitrator instance:
+
+```
+./haveno-desktop --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=4444 --appName=haveno-XMR_STAGENET_arbitrator --daoActivated=false --apiPassword=apitest --apiPort=9998
+```
+
+The (temporary) Haveno user interface will open. Once Haveno will be synced:
+
+3. click on the *Account* tab and press `cmd+n`. Confirm the registration of the arbitrator.
+4. From the *Account* tab press `cmd+d` and confirm the registration of the mediator.
+
+Make sure to register arbitrator and mediator, without them it's not possible to initiate a trade on the PoC.
+
+5. Run Alice
+
+```
+./haveno-desktop --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=5555 --appName=haveno-XMR_STAGENET_Alice --daoActivated=false --apiPassword=apitest --apiPort=9999
+```
+
+6. Run Bob
+
+```
+./haveno-desktop --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=6666 --appName=haveno-XMR_STAGENET_Bob  --daoActivated=false --apiPassword=apitest --apiPort=10000
+```
+
+## 5. Get yourself some stagenet coins
+
+You can mine from you `monerod` instance or contact ErCiccione on Matrix and give him a stagenet address (`@ErCiccione:haveno.network`). Note that the address is not yet in the GUI, but it's visible from the terminal from which you run your Haveno PoC instance.
+
+Mining is the suggested method to receive stagenet coins. The difficulty is locked to 10, so mining would be very fast. Go to your Monero instance and run `start_mining ADDRESS 1`. Monerod will start mining stagenet coins on your device using one thread. Replace `ADDRESS` with the address of Alice first, and then Bob's. 
+
+Remember to send funds to both Alice and Bob.
+
+## 6. Start trading
+
+Now you can open and take offers using the Haveno PoC (which is still mostly Bisq in the user interface. The redesign is in progress.)
```

Comment:
The application needs to be opened before knowing the address to send funds to.

The instructions need to be clear step-by-step or we'll spend time answering setup problems.

Reply:
> The application needs to be opened before knowing the address to send funds to.

The application was open in step 5 (`5. Run Haveno`).